cpp/tests/quantiles/quantile_test.cpp

```diff
@@ -448,6 +448,28 @@ TYPED_TEST(QuantileUnsupportedTypesTest, TestMultipleElements)
   EXPECT_THROW(cudf::quantile(input, {0}), cudf::logic_error);
 }
 
+struct QuantileDictionaryTest : public BaseFixture {
+};
+
+TEST_F(QuantileDictionaryTest, TestValid)
+{
+  fixed_width_column_wrapper<int32_t> col{1, 2, 3, 4, 5, 6, 7, 8, 9, 10};
+  fixed_width_column_wrapper<int32_t> indices{0, 2, 4, 6, 8, 1, 3, 5, 7, 9};
+
+  auto result = cudf::quantile(col, {0.5}, cudf::interpolation::LINEAR);
```

Comment:
I might be missing something, col seems to be normal `int32` column, how does it test dictionary?